sos/report/plugins/memory.py

```diff
@@ -26,12 +26,15 @@ def setup(self):
             "/proc/slabinfo",
             "/proc/pagetypeinfo",
             "/proc/vmallocinfo",
+            "/sys/devices/system/memory",
+            "/sys/kernel/mm/hugepages",
             "/sys/kernel/mm/ksm",
             "/sys/kernel/mm/transparent_hugepage/enabled"
         ])
         self.add_cmd_output("free", root_symlink="free")
         self.add_cmd_output([
             "free -m",
+            "free -h",
```

Comment:
We already get 2 copies of `free` output, I don't think we need a third.

Reply:
Thanks Jake,
Will update your comments and send v2 patchset